ci/runGDPR.yml

```diff
@@ -0,0 +1,37 @@
+resources:
+- name: schedule
+  type: time
+  source:
+    start: 2:00 AM
+    stop: 5:00 AM
+    location:   
```

Comment:
Do we need `location:`? It is not defined yet.

Reply:
@adityapahuja Location is used by concourse, not declaring a value means it will default to UTC. I don't know what would happen if I don't include it...

Reply:
According to the Concourse page (https://github.com/concourse/time-resource#source-configuration), `location` is an optional field. It is fine to remove the field because it is defaulted to UTC. There are other optional fields e.g. interval and days.

Reply:
@adityapahuja great, resolved as suggested.